test/unit/test_lld.py

```diff
@@ -0,0 +1,14 @@
+from scripts.test.shared import WASM_EMSCRIPTEN_FINALIZE, run_process
```

Comment:
Would this be better named `test_finalize.py`?

Can you add a copyright header?

Reply:
I am just using the `lld` name because all the other finalize tests are under `test/lld`.

As for copyright header, none of the other unit test files have it.

Reply:
I would call it `test_finalize`.  The other tests in test/lld are there because they are actually generated by lld itself.  In this case you can hard-crafting the input.

Reply:
Done.